test_app/tests/tests_views/tests_filter_views.py

```diff
@@ -50,3 +50,18 @@ def test_search_body_malformed(self):
         search = "potato"
         response = self.client.post(url, data=search, content_type="application/text")
         self.assertEqual(status.HTTP_415_UNSUPPORTED_MEDIA_TYPE, response.status_code)
+
+    def test_search_body_ordered(self):
+        teachers = []
+        for i in range(10, 20):
+            teachers.append(TeacherFactory(id=i, is_ghost=False, picture=None))
```

Comment:
The `id` could be omitted. It seems irrelevant for your test

Reply:
I am sorting by `id` in the test, so that's why I'm specifically including its value.